operator/v1/types_network.go

```diff
@@ -79,9 +79,10 @@ type NetworkSpec struct {
 	// +listMapKey=name
 	AdditionalNetworks []AdditionalNetworkDefinition `json:"additionalNetworks,omitempty"`
 
-	// disableMultiNetwork specifies whether or not multiple pod network
-	// support should be disabled. If unset, this property defaults to
-	// 'false' and multiple network support is enabled.
+	// disableMultiNetwork enables multiple network support when the OVN-Kubernetes Container Network Interface (CNI)
+	// plugin sets the parameter to the default value of 'false'. During cluster installation only, you can use a 
+	// third-party CNI, such as Multus CNI, to set the disableMultiNetwork parameter to 'true', and this setting disables
+	// multiple pod network support. Red Hat does not support third-party CNIs.
```

Comment:
I think we need a few tweaks...

My I suggest...

```
// disableMultiNetwork when set to true at cluster install will disable the installation of components which enable pod multinetworking, especially utilizing Multus CNI and network-attachment-definition custom resources. The default value is false.
// This option may potentially be useful for installation of third party CNI plugins, however, this is not supported by Red Hat.
// Setting this option after cluster install has no effect.
```

Happy to take a mix of the two, but, we should definitely remove: `you can use a third-party CNI, such as Multus CNI, to set the disableMultiNetwork parameter to 'true'`. As setting this to true by the user in the install-config will install Multus, might be the confusion.